hangman.rb

```diff
@@ -0,0 +1,263 @@
+              "|     |      ",
+              "|     |",
+              "| ____|___"
+            ],
+                                    [
+              "|     _________",
+              "|     |/      |",
+              "|     |      (_)",
+              "|     |      "+ "\\".colorize(:cyan) + "|",
+              "|     |       |",
+              "|     |      ",
+              "|     |",
+              "| ____|___"
+            ],
+                                                [
+              "|     _________",
+              "|     |/      |",
+              "|     |      (_)",
+              "|     |      \\|" + "/".colorize(:blue),
+              "|     |       |",
+              "|     |      ",
+              "|     |",
+              "| ____|___"
+            ],
+                                                [
+              "|     _________",
+              "|     |/      |",
+              "|     |      (_)",
+              "|     |      \\|/ ",
+              "|     |       | ",
+              "|     |      " + "/ ".colorize(:magenta),
+              "|     |",
+              "| ____|___"
+            ],
+                                                [
+              "|     _________",
+              "|     |/      |",
+              "|     |      (_)",
+              "|     |      \\|/",
+              "|     |       |",
+              "|     |      / " + "\\ ".colorize(:light_black),
+              "|     |",
+              "| ____|___"
+]]
+
+    start
+  end
+
+  def pick_word
+    word_list = %w{cat dog egg horse donkey elephant pig monkey iguana cow goat unigoat eggplant mouse dragon phoenix}
+    random_word = word_list.sample
+    return random_word
+  end
+
+  def start
+    puts "Welcome to Hangman."
+    puts "Please type your whole word or letter guess. No numbers, punctuation, or spaces."
+    # uncomment the following line for aid in development
+    puts "Answer: #{@game_word}"
+
+    @now_playing = true
+    puts "To exit, type: quit."
+    draw_tree(0)
+   
+    draw_slots
+    draw_wrong_guesses
+
+    play
+  end
+
+  def play
+    while @now_playing
+      guess = get_guess
+      check_guess(guess)
+      update_board
+      check_win
+    end
+  end
+
+  # ---------------------------------
+  # Guess Methods
+
+  def get_guess
+    input = gets.chomp.strip.downcase
+    # allow user to exit game
+    if input == "quit" || input == "exit"
+      exit
+    else
+      guess = validate_input(input)
+    end
+    return guess
+  end
+
+  def validate_input(input)
+    valid = true
+
+    # change valid to false if input string includes anything but a letter
+    input.split("").each do |letter|
+        if !(("a".."z").include?(letter))
+          valid = false
+        end
+      end
+
+    if valid
+      return input
+    else
+      puts "Error:".colorize(:red) + " Please only use letters. No numbers, punctuation, or spaces."
+      get_guess
+    end
+  end
+
+  def check_guess(guess_letter)
```

Comment:
Since you have two distinct paths here: a word or a letter, it might make sense to separate those out into two additional check_word_guess and check_letter_guess or something like that.